src/api-db-migrations/20230808120700__add-disabledCirclesAddressColumn.sql

```diff
@@ -0,0 +1,45 @@
+-- migrate:up
+ALTER TABLE "Profile"
+  ADD COLUMN "disabledCirclesAddress" TEXT;
+
+CREATE OR REPLACE PROCEDURE sp_DisableProfileAndShop(address varchar(100))
+LANGUAGE plpgsql
+AS $$
+BEGIN
+  -- Copy circlesAddress to disabledCirclesAddress
+  UPDATE
+    "Profile"
+  SET
+    "disabledCirclesAddress" = "circlesAddress"
+  WHERE
+    "circlesAddress" = address;
+  -- Set disabled status
+  UPDATE
+    "Profile"
+  SET
+    "status" = 'disabled'
+  WHERE
+    "circlesAddress" = address;
+  -- Remove CirclesAddress
+  UPDATE
+    "Profile"
+  SET
+    "circlesAddress" = NULL
+  WHERE
+    "circlesAddress" = address;
+  -- Disable the Shop
+  UPDATE
```

Comment:
This is not really how the memberships have been designed in the first place. Still, it works like we use it right now. Maybe add a filter to the "isAdmin" column if we decide to revive menberships later.

Reply:
Done.